portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase

```diff
@@ -84,7 +84,7 @@ definition {
 		AssertCssValue(
 			locator1 = "//ul[contains(@id,'controlMenu')]",
 			locator2 = "list-style",
-			value1 = "none outside none");
+			value1 = "outside none none");
 
 		takeScreenshot();
 
```

Comment:
LPS-63100 is only for RefreshWebContent test, you're making changes to RefreshHomePage in this commit... you need to split the changes off to a different commit.

Reply:
Please ask @manoelcyreno  or @zhangyan0701 if need clarification on how to do this.